content/blog/2023/10/camel-k-2-1/index.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Camel K 2.1"
+date: 2023-10-20
+draft: false
+authors: [squakez]
+categories: ["Releases", "Camel K", "Roadmap"]
+preview: "What's new in Camel K 2.1!"
+---
+
+Apache Camel community is happy to announce the general availability of **Camel K 2.1**. We have worked on the introduction of a lot of new exciting feature that will further **simplify the deployment and management of Camel application on Kubernetes**. Let's see what you should expect from this release.
+
+## Default Camel 4 runtime
+
+As **Camel 4** is out since some time already, we've moved our default runtime to Camel K Runtime version 3.2.0, which is based on Camel Quarkus 3.2.0 and Camel 4. From now on, by default you will use this runtime version.
+
+## More enterprise ready build
+
+One of the main goal of 2023 roadmap was to make our **build system "enterprise ready"**. In this release we've worked a lot in that direction.
+
+The pipeline concept we have introduced in version 2.0 is now more complete with the split of the build tasks into a proper `build` and a `package` which is in charge to prepare the context for a following publish with one of the strategies chosen by the user. The result is illustrated in the following diagram:
+
+![Camel K Pipeline tasks](camel_k_pipeline.png)
+_Camel K Pipeline tasks_
+
+This gives you more flexibility by letting you customize the build before the binaries are taken by the package task. Beside this change we have introduced a lot of advanced configuration in the Builder trait. Each custom task you will define, can be configured with limits and requests resources: `builder.tasks-request-cpu`, `builder.tasks-request-memory`, `builder.tasks-limit-cpu`, `builder.tasks-limit-memory`.
+Another interesting option we have introduced, was to let the user disable the Incremental Image Build option via `builder.incremental-image-build`. It is not advisable, but there could be situations were you don't want your Integration inherit from existing IntegrationKits.
+
+Always related to the builder, you will find now the presence of `builder.maven-profile` option. With this one you can provide a customized [Maven profile](https://maven.apache.org/guides/introduction/introduction-to-profiles.html#profiles-in-poms) to your build. This is cool if you want to influence the build phase in any manner.
```

Comment:
This one got changed to a multi-valued field `builder.maven-profiles`. That mean you can provide multiple maven profiles